test/services/helpers/services/index.js

```diff
@@ -1,37 +1,39 @@
-const teams = require('./teams');
-const testSystem = require('./testSystem');
-const login = require('./login');
-const classes = require('./classes');
-const users = require('./users');
-const courses = require('./courses');
 const accounts = require('./accounts');
-const roles = require('./roles');
-const schools = require('./schools');
-const schoolGroups = require('./schoolGroups');
-const years = require('./years');
+const classes = require('./classes');
 const consents = require('./consents');
+const courses = require('./courses');
 const datasources = require('./datasources');
+const files = require('./files');
 const homeworks = require('./homeworks');
-const submissions = require('./submissions');
 const lessons = require('./lessons');
+const login = require('./login');
+const roles = require('./roles');
+const schoolGroups = require('./schoolGroups');
+const schools = require('./schools');
 const storageProviders = require('./storageProviders');
+const submissions = require('./submissions');
+const teams = require('./teams');
+const testSystem = require('./testSystem');
+const users = require('./users');
+const years = require('./years');
```

Comment:
The changes here are mostly to sort them alphabetically. There are a few more of these lists below. The only new entity is `files`.